app/domain/urls.ts

```diff
@@ -45,6 +45,16 @@ const localeAwareUrls: Urls = {
     api: '/api',
     apiDocs: '/api-docs',
     statistics: '/statistics'
+  },
+  sk: {
+    submitReport: '/',
+    profile: '/zdravotny-stav',
+    privacyPolicy: '/ochrana-udajov',
+    map: '/mapa',
+    contributors: '/prispievatelia',
+    api: '/api',
+      apiDocs: '/api-docs',
```

Comment:
Not critical, but the indentation is wrong

Reply:
yeah, nice catch, thank you!